src/main/java/com/shrek/task00001/Main.java

```diff
@@ -9,10 +8,74 @@
  */
 public class Main {
     public String getWinner(String[] runners, int[] times) {
```

Comment:
Great! It works. And now lets improve your code. Try to find solution without sorting. It will be just one cycle.